content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -1,285 +1,268 @@
 
-```sh
-pip install opentelemetry-propagator-b3
-```
+Automatic instrumentation captures telemetry at the edges of your systems, such
+as inbound and outbound HTTP requests, but it doesn't capture what's going on in
+your application. For that you'll need to write some [manual
+instrumentation]({{< relref"manual" >}}). Here's how you can easily link up
+manual instrumentation with automatic instrumentation.
 
-Following the installation of the package containing the b3 propagator,
-configure the propagator as follows:
+First, modify `app.py` to include code that initializes a tracer and uses it to
+create a trace that's a child of the one that's automatically generated:
 
 ```python
-from opentelemetry.propagate import set_global_textmap
-from opentelemetry.propagators.b3 import B3Format
-
-set_global_textmap(B3Format())
-```
-
-## Use the OpenTelemetry Collector for traces
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
 
-Although it’s possible to directly export your telemetry data to specific
-backends, you might have more complex use cases such as the following:
+from random import randint
+from flask import Flask, request
 
+provider = TracerProvider()
+trace.set_tracer_provider(provider)
+tracer = trace.get_tracer(__name__)
 
-* A single telemetry sink shared by multiple services, to reduce overhead of
-  switching exporters.
+app = Flask(__name__)
 
+@app.route("/roll")
+def roll():
+    sides = int(request.args.get('sides'))
+    rolls = int(request.args.get('rolls'))
+    return roll_sum(sides,rolls)
 
-* Aggregating traces across multiple services, running on multiple hosts.
 
-To enable a broad range of aggregation strategies, OpenTelemetry provides the
-[opentelemetry-collector](https://github.com/open-telemetry/opentelemetry-collector).
-The Collector is a flexible application that can consume trace data and export
-to multiple other backends, including to another instance of the Collector.
+def roll_sum(sides, rolls):
+    # This creates a new trace that's the child of thw current one, if it exists
+    with tracer.start_as_current_span("roll_sum"):  
+        sum = 0
+        for r in range(0,rolls):
+            result = randint(1,sides)
+            sum += result
+        return  str(sum)
+```
 
-Start the Collector locally to see how the Collector works in practice. Write
-the following file:
+Now run the app again:
 
-```yaml
-# /tmp/otel-collector-config.yaml
-receivers:
-    otlp:
-        protocols:
-            grpc:
-            http:
-exporters:
-    logging:
-        loglevel: debug
-processors:
-    batch:
-service:
-    pipelines:
-        traces:
-            receivers: [otlp]
-            exporters: [logging]
-            processors: [batch]
+```console
+$ opentelemetry-instrument --traces_exporter console flask run
 ```
 
-Then start the Docker container:
+You'll now see two spans in the trace emitted to the console, and the one called
```

Comment:
Add that you need to send a web request to the server, then you will see the spans.